CSharp/DevVmPowershell/Helpers.Tests.Integration/TestConstants.cs

```diff
@@ -3,8 +3,8 @@
 	public class TestConstants
 	{
 		public const string TEST_APPLICATION_NAME = "Production";
-		public const string RELATIVITY_INSTANCE_NAME = "172.24.170.148";
-		public const string SAMPLE_DATA_GRID_WORKSPACE_NAME = "Sample Data Grid Workspace";
+		public const string RELATIVITY_INSTANCE_NAME = "172.17.208.218";
+		public const string SAMPLE_DATA_GRID_WORKSPACE_NAME = "Sample Workspace";

```

Comment:
can we update the variable name to remove datagrid as its updated?

Reply:
leaving it as is